shared/source/os_interface/linux/driver_info_linux.cpp

```diff
@@ -7,19 +7,40 @@
 
 #include "shared/source/os_interface/linux/driver_info_linux.h"
 
+#include "shared/source/os_interface/linux/drm_neo.h"
+#include "shared/source/os_interface/linux/os_interface.h"
+#include "shared/source/helpers/debug_helpers.h"
 #include "shared/source/helpers/hw_info.h"
 
 namespace NEO {
 
-DriverInfo *DriverInfo::create(const HardwareInfo *hwInfo, OSInterface *osInterface) {
+DriverInfo *DriverInfo::create(const HardwareInfo *hwInfo, const OSInterface *osInterface) {
+    PhysicalDevicePciBusInfo pciBusInfo;
+    if (osInterface) {
+        auto osInterfaceImpl = osInterface->get();
+        DEBUG_BREAK_IF(osInterfaceImpl == nullptr);
+
+        if (osInterfaceImpl) {
+            auto drm = osInterface->get()->getDrm();
+            DEBUG_BREAK_IF(drm == nullptr);
+
+            if (drm) {
```

Comment:
These checks are not necessary and should not happen, if in doubt add UNRECOVERABLE_IF (instead of checking in if statement)

Reply:
The check has been replaced with `UNRECOVERABLE_IF()`.